src/main/scala/org/clulab/habitus/utils/JsonPrinter.scala

```diff
@@ -48,11 +48,12 @@ class JsonPrinter(outputFilename: String) extends Printer {
         ("sentenceText" -> sentenceText) ~
         ("inputFilename" -> inputFilename)
 
-    // make an obj from key-value pairs from the context attachment
-    val argJObject: JObject = toJObject(mention.attachments.head.asInstanceOf[Context].getArgValuePairs())
-
-    for (value <- argJObject.obj) {
-      jObject = jObject ~ (value._1 -> value._2)
+    val context = mention.attachments.headOption
```

Comment:
In a different iteration, I may include a method to perform this conversion and then add it to some shared superclass of JsonPrinter and JsonlPrinter along with the toJObject and anything else they share.